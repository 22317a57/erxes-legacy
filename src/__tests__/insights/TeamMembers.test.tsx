import { shallow } from 'enzyme';
import React from 'react';

import TeamMembers from '../../modules/insights/components/TeamMembers';

describe('TeamMembers component', () => {
  const defaultProps = {
    datas: [
      {
        x: 'length',
        y: 0,
        graph: []
      }
    ],
    loading: false
  };

  test('renders TeamMembers successfully', () => {
    shallow(<TeamMembers {...defaultProps} />);
  });
});
