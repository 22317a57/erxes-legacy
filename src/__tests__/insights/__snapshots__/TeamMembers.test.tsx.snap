// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamMembers component snapshot matches 1`] = `
<div
  className="row"
>
  <div
    className="col-sm-6"
  >
    <div
      className="sc-fOICqy gWshsg"
    >
      <div
        className="sc-hzDEsm kzLTOg"
      >
        <a>
          <img
            src="/images/avatar-colored.svg"
          />
        </a>
        <span
          className="sc-jeCdPy fHVrmW"
        />
      </div>
      <div
        className="sc-iGPElx hgMnov"
      >
        <div
          className="sc-cugefK iITMyU"
        >
          <i
            className="icon-piechart sc-bwzfXH hpxbGw"
            icon="piechart"
          />
          <span>
            There is no data
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
